src/main/java/no/sikt/nva/pubchannels/model/CreateJournalRequest.java

```diff
@@ -5,18 +5,46 @@
 
 public class CreateJournalRequest {
     private static final String NAME_FIELD = "name";
+    private static final String PISSN_FIELD = "pissn";
+    private static final String EISSN_FIELD = "eissn";
+    private static final String URL_FIELD = "url";
 
     @JsonProperty(NAME_FIELD)
     private final String name;
+    @JsonProperty(PISSN_FIELD)
+    private final String pissn;
+    @JsonProperty(EISSN_FIELD)
+    private final String eissn;
+    @JsonProperty(URL_FIELD)
+    private final String url;
 
     @JsonCreator
     public CreateJournalRequest(
-            @JsonProperty(NAME_FIELD) String name
+            @JsonProperty(NAME_FIELD) String name,
+            @JsonProperty(PISSN_FIELD) String pissn,
+            @JsonProperty(EISSN_FIELD) String eissn,
+            @JsonProperty(URL_FIELD)String url
     ) {
         this.name = name;
+        this.pissn = pissn;
```

Comment:
I see that you trim texts when validating input. To ensure we pass on good data to the external API, why not trim input here?

Reply:
I will make this more presentable by changing pissn to printIssn and eissn to onlineIssn.